src/components/Main.js

```diff
@@ -1,13 +1,19 @@
 import React from "react";
-import Map from '../components/Map';
+import PropTypes from "prop-types";
+import Map from "../components/Map";
 
+const Main = (props) => {
+  const { data } = props;
```

Comment:
```suggestion
const Main = ({data}) => {
```
This simplifies the code by unpacking the props and assigning to a variable within the parameter.

Reply:
Nice one, Jen!